src/main/java/com/jfrog/ide/idea/ui/webview/model/Cve.java

```diff
@@ -0,0 +1,64 @@
+package com.jfrog.ide.idea.ui.webview.model;
+
+import java.io.Serializable;
+
+public class Cve implements Serializable {
+    String id;
+    String cvssV2Score;
+    String cvssV2Vector;
+    String cvssV3Score;
+    String cvssV3Vector;
+    ApplicableDetails applicableData;
+
+    public Cve() {
+    }
+
+    public Cve(String id, String cvssV2Score, String cvssV2Vector, String cvssV3Score, String cvssV3Vector) {
+        this.id = id;
+        this.cvssV2Score = cvssV2Score;
+        this.cvssV2Vector = cvssV2Vector;
+        this.cvssV3Score = cvssV3Score;
+        this.cvssV3Vector = cvssV3Vector;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+
+    public String getCvssV2Vector() {
```

Comment:
Let's add this in all places whereby we have an unused methods with a justified reason
```suggestion
    @SuppressWarnings("unused")
    public String getCvssV2Vector() {
```